### PR TITLE
Simplify phrasing and diction. Fix grammatical errors.

### PR DESCRIPTION
**Pull request by PatrickBM**
This includes no substantive content changes.

### CODE REVIEW

File: advanced/index.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ Explore the structure and way change is preserved in Git.
 ---
 
 {% capture lab %}
-1. Understand the way Git preserves history and data objects
-2. Utilize shorthand for navigation commit history
+1. Understand the way Git preserves data
```
Comment (randomecho) [inline advanced/index.md]: The history of that data, where it's been and what it was at certain points in time, is one of the fundamental aspects of Git. It's not just about preserving data, but tracking its evolution.

File: advanced/index.md [hunk 2]
```diff
@@ -401,15 +401,15 @@ Rebase and reorder existing commits for improved historical context.
...
+2. Understand the side effects of rewriting history
+3. Rewrite history interactively with `rebase -i`
+4. Resolve merge conflicts during a rebase
 {% endcapture %}{% include lab %}
 
 
 
 ### Details
-Adjusting, rewriting and crafting existing version control history is often completed with `rebase`. The most important aspect to note is that `rebase` is distinct from `merge` and not the same, despite their similarities.
+Use `rebase` to ddit existing commit history. But pay close attention to how it differs from `merge`
```
Comment (randomecho) [inline advanced/index.md]: Should be "edit" instead of "ddit"

File: advanced/index.md [hunk 3]
```diff
@@ -94,14 +94,14 @@ $ git log -G [regex-pattern-in-patch]
 $ git log --grep=[regex-in-message]
 ```
 
-Filtering by file state, *added* (A), *modified* (M), or *deleted* (D), also narrows what change requires assessment.
+Filtering by file state, i.e. *added* (A), *modified* (M), or *deleted* (D):
```
Comment (randomecho) [inline advanced/index.md]: Perhaps we can add back in what/why we would want to filter by state, so that the sentence gives some context/reason to the action.

Would this not be a case for "e.g." as they are options to choose between, but not actions to follow as-is?

File: advanced/index.md [hunk 4]
```diff
@@ -71,15 +71,15 @@ Historical commit points are simple to discuss using *commitish* rather than hex
 Shorthand              | Explanation
 ---------------------- | -----------
 `HEAD`                 | Current commit
-`HEAD^^`               | Second parent of current commit
-`HEAD~2`               | Two commits from current commit
+`HEAD^2`               | Second parent of current commit (assumes current commit the result of a merge)
+`HEAD~2`               | First parent (commit) of the first parent of the current commit (different than above!)
```
Comment (randomecho) [inline advanced/index.md]: Found this a little harder to read given the repetition of "first parent", a little like that "party in the part of the first part".

File: advanced/index.md (hunk 1 above)
Comment (PatrickBM) [inline advanced/index.md]: I agree. My thought was that the act of preserving is the fundamental activity, and that how one preserves history is encapsulated in the story of how one preserves data. But I'm happy to revert this change if the new sentence is too general (or otherwise worse than the original).

File: advanced/index.md (hunk 3 above)
Comment (PatrickBM) [inline advanced/index.md]: Good point, will re-add a phrase motivating the use of these flags.

E.g. means "for example," while i.e. stands for "that is to say." A, M, and D are the only three states available (should we add untracked?), so the reference to state implies one of those three options -- there are no other example states.

File: advanced/index.md (hunk 4 above)
Comment (PatrickBM) [inline advanced/index.md]: Yeah, I wasn't totally happy with this either. First and second describe both what parent one's discussing for a single merge commit (i.e. first is the branch you were on, second is the branch you merged in). But of course they're also used to say how far back in history one goes. Totally open to suggestions.

File: advanced/index.md [hunk 5]
```diff
@@ -430,12 +430,13 @@ $ git config pull.rebase true
 Resolving a conflict during a rebase requires intervention, and then signaling that the conflict is resolved.
 
 ```shell
-$ git add [conflicting-file]
+# fix conflicts
+$ git add [conflict-free file]
```
Comment (brntbeer) [inline advanced/index.md]: really like this subtle differentiation

**Comment by brntbeer** [comment]
I like a lot of the language cleanup, seems like there may still be some comments to decide on, but it's real good!

File: advanced/index.md (hunk 1 above)
Comment (jordanmccullough) [inline advanced/index.md]: My suggestion as a happy medium would be:

> 1. Understand the way Git preserves history and data

This removes the need to concern readers with what "objects" is from a objectives list standpoint.

File: advanced/index.md (hunk 4 above)
Comment (jordanmccullough) [inline advanced/index.md]: Happy to see tune-ups on these descriptions happening. If we'd like to be consistent with Git documentation, and even the Pro Git book, I'd suggest we use the same vocabulary from those docs here as well.

Suggestions:

> `HEAD^2` Second parent if a merge commit, otherwise second ancestor.

> `HEAD~2` Second ancestor of the current commit

Comment (PatrickBM) [inline advanced/index.md]: > If we'd like to be consistent with Git documentation, and even the Pro Git book,

All for this. Ironically, I actually stole that "parent of parent" language from Pro Git -- though I much prefer your suggestions.

**Comment by PatrickBM** [comment]
@github/training Any further suggestions or edits to make or should we :ship: this? (I don't want to merge my own changes.)

**Comment by brntbeer** [comment]
If you dont want to merge the changes, i will! Nice additions @PatrickBM
